components/softdevice/s115/doc/s115_9.0.0-4.prototype_release-notes.rst

```diff
@@ -0,0 +1,96 @@
+s115_9.0.0-4.prototype
+======================
+
+This version is a major release, providing support for nRF54L series devices.
```

Comment:
version is 9.x.x.x, the major version has not changed? And nrf54l was already supported?

Reply:
Files in `components/softdevice` are not for review and excluded from compliance checks since they are build outputs. If you have any improvements you want done, open a PR in the dragoon repo.